perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py

```diff
@@ -55,18 +54,20 @@
 cassandra_stress:
   description: Benchmark Cassandra using cassandra-stress
   vm_groups:
-    default:
+    cassandra:
+      vm_spec: *default_single_core
+      disk_spec: *default_500_gb
```

Comment:
Why has the size been hard coded for hte default data disk?  Why not call this "*default_data_disk"?